benchmark-defs/dartagnan.xml

```diff
@@ -43,5 +43,14 @@
     <option name="-w"></option>
     <option name="-i"></option>
   </tasks>
+  <tasks name="NoDataRace-Main">
+    <includesfile>../sv-benchmarks/c/NoDataRace-Main.set</includesfile>
+    <propertyfile>../sv-benchmarks/c/properties/no-data-race.prp</propertyfile>
+    <option name="-cat">cat/svcomp.cat</option>
+    <option name="-races"></option>
+    <option name="-incrementalSolver"></option>
+    <option name="-w"></option>
+    <option name="-i"></option>
+  </tasks>
```

Comment:
Since I'll need to add something similar for Goblint, I'm wondering if this shouldn't be in `<rundefinition name="sv-comp21_prop-reachsafety">` but instead in something new like `<rundefinition name="sv-comp21_prop-nodatarace">` because the property is new and different. I'm not sure what difference this makes though.

Maybe @dbeyer can clear this up so we all add our NoDataRace stuff consistently.